src/test/java/org/jitsi/meet/test/PSNRTest.java

```diff
@@ -196,6 +207,20 @@ public void testPSNR()
                 File outputFrameFile = new File(outputFrame);
                 outputFrameFile.delete();
             }
+            float averagePsnr = totalPsnr / framesCount;
```

Comment:
Would we also want to fail the test if the average is below a certain threshold (the per-frame threshold is very very low)?

Reply:
we could, but i think the per-frame one will be even more sensitive, no? (unless you mean having the average required to be an even higher value?)